### ISSUE TITLE
Panics on first run

### ISSUE THREAD

**Issue by TheScreechingBagel** [ayntgl/discordo#53]
Installed from the AUR with yay
packages all updated

when running `discordo` this outputs and exits:
```
panic: open /home/bagel/.config/discordo/config.json: no such file or directory

goroutine 1 [running]:
github.com/rigormorrtiss/discordo/util.LoadConfig()
	/home/bagel/.cache/yay/discordo-git/src/discordo/util/config.go:44 +0x296
main.main()
	/home/bagel/.cache/yay/discordo-git/src/discordo/main.go:28 +0x2b
```

**Comment by ayntgl** [comment]
The [`discordo-git`](https://aur.archlinux.org/packages/discordo-git/) AUR package is fairly outdated, you should try building Discordo from the source. You can find the instructions on how to build it from the source in the [README](https://github.com/ayntgl/discordo/blob/main/README.md). Let me know if the issue persists after building it from the source.

**Comment by TheScreechingBagel** [comment]
built from source, running `discordo` gives a similar error:
```
panic: open /home/bagel/.config/discordo/config.json: no such file or directory

goroutine 1 [running]:
github.com/rigormorrtiss/discordo/util.LoadConfig()
	/home/bagel/Desktop/discordo/util/config.go:46 +0x374
main.main()
	/home/bagel/Desktop/discordo/main.go:28 +0x2b
```

**Comment by ayntgl** [comment]
@TheScreechingBagel, this issue is fixed by #54, let me know if you face any other problems.
